src/main/js/helpers/region.js

```diff
@@ -17,6 +17,70 @@ import utils from "./utils";
  * @alias module:region
  */
 
+/*
+ * global array to store the region if same
+ */
+const globalregionData = [];
+
+/**
+ * Compares the region to check is region is same for Line graph
+ * @private
+ * @param {Object} region - Region to be shown within graph
+ * @returns {Boolean} - returns true or false
+ */
+const compareRegionDataLine = (region) => {
+    if (globalregionData.length > 0) {
+        if (
+            globalregionData[0].start === region.start &&
+            globalregionData[0].end === region.end &&
+            globalregionData[0].axis === region.axis
+        ) {
+            return true;
+        } else {
+            return false;
+        }
+    }
+};
+
+/**
+ * Compares the region to check is region is same for PairedResult graph
+ * @private
+ * @param {Object} region - Region to be shown within graph
+ * @param { Object } graphConfig - graph propertiy needed to check the isRegionSame property
+ * @returns {Boolean} - returns true or false
+ */
+const compareRegionDataPaired = (region, graphConfig) => {
+    if (
+        globalregionData.length > 0 &&
+        graphConfig.isPairedDataProper === true
+    ) {
+        if (
+            globalregionData[0].start === region.start &&
+            globalregionData[0].end === region.end &&
+            globalregionData[0].axis === region.axis
+        ) {
+            return true;
+        } else {
```

Comment:
Unnecessary use of else block in - line 63 and 66, return directly

Reply:
isRegionSame me be set true for one load and for the second load the region is not same for pairedResult then we have to return false.